src/components/transactions/modals/ClaimWRBTCModal.vue

```diff
@@ -160,6 +169,7 @@ export default {
   },
   mounted() {
     this.handleChangeClaimType({ target: { value: this.claimTypes.STANDARD } })
+    this.swapRbtcProxyAddress = getProjectAddress('SWAP_RBTC_PROXY_V1')
```

Comment:
this could be simply `this.sharedState.currentConfig.swapAddress` if you create the config the same way the "this.sharedState.currentConfig.relayer" is set.